client/js/components/show-hide.js

```diff
@@ -0,0 +1,42 @@
+const showHide = (state) => {
+  const el = state.el;
+  const activeClass = state.activeClass || 'is-active';
```

Comment:
The element containing an `aria-expanded` attribute is a sibling of the expanded/collapsed item. I thought about doing this with the adjacent sibling selector, but concluded it was more fragile than relying on a parent with a state class.